dockerfile

```diff
@@ -17,6 +17,19 @@ RUN npm install --legacy-peer-deps && \
 
 
 FROM node:14-bullseye-slim AS release
+
+ARG BUILD_DATE
```

Comment:
Please use default value. For example in bash: `date -u +'%Y-%m-%dT%H:%M:%SZ'`

(from https://medium.com/@chamilad/lets-make-your-docker-image-better-than-90-of-existing-ones-8b1e5de950d)

Reply:
In the PR, the date is passed with `docker build --build-arg` from  `package.json` what is literally like in the example you gave. There are no way to calculate the `ARG` in dockerfile or pass `ENV` from one building step to another. You can only set constant value or use another [variable substitution](https://vsupalov.com/docker-arg-env-variable-guide/) (`ARG`→`ENV`).
So, exactly like in your example, there is no default `BUILD_DATE` to set here. What I know this is how docker image building process works, and it's not a bug or incorrect coding. Correct me if I'm [wrong](https://github.com/louislam/uptime-kuma/pull/442#discussion_r712449023) and if there is a way to set and pass to `LABEL` a variable during build process.